packages/blade/src/components/ListView/docs/ListView.stories.tsx

```diff
@@ -0,0 +1,1134 @@
+  );
+};
+
+const TabNavItemLink = React.forwardRef<
+  HTMLAnchorElement,
+  Omit<TabNavItemProps, 'as'> & {
+    activeOnLinks?: string[];
+  }
+>((props, ref) => {
+  return <TabNavItem ref={ref} {...props} isActive={props.href === '/home'} />;
+});
+
+const SideNavExample = ({
+  isOpen,
+  onDismiss,
+}: Pick<SideNavProps, 'isOpen' | 'onDismiss'>): React.ReactElement => {
+  return (
+    <SideNav isOpen={isOpen} onDismiss={onDismiss} position="absolute">
+      <SideNavBody>
+        <SideNavLink icon={HomeIcon} title="Home" href="/home" as={Link} />
+        <SideNavLink icon={LayoutIcon} title="L2 Trigger" href="/l2-item" as={Link}>
+          <SideNavLevel>
+            <SideNavLink title="L2 Item" href="/l2-item" as={Link} />
+            <SideNavLink title="L2 Item 2" href="/l2-item-2" as={Link} />
+            <SideNavLink title="L3 Trigger" href={'/l3-item-2'} as={Link}>
+              <SideNavLevel>
+                <SideNavLink title="L3 Item" href="/l3-item" as={Link} />
+                <SideNavLink title="L3 Item 2" href="/l3-item-2" as={Link} />
+              </SideNavLevel>
+            </SideNavLink>
+          </SideNavLevel>
+        </SideNavLink>
+
+        <SideNavSection title="Products" maxVisibleItems={2}>
+          <SideNavLink icon={PaymentGatewayIcon} title="Gateway" href="/gateway" as={Link} />
+          <SideNavLink icon={PaymentLinkIcon} title="Links" href="/links" as={Link} />
+          <SideNavLink icon={PaymentPagesIcon} title="Pages" href="/pages" as={Link} />
+          <SideNavLink icon={PaymentButtonIcon} title="Button" href="/button" as={Link} />
+        </SideNavSection>
+      </SideNavBody>
+    </SideNav>
+  );
+};
+const ListViewTemplate: StoryFn<typeof ListView> = (args) => {
+  const [value, setSelectedValue] = React.useState<string[]>([]);
+  const handleOnClick = (name: string): void => {
+    if (value.includes(name)) {
+      setSelectedValue(value.filter((val) => val !== name));
+    } else {
+      setSelectedValue([...value, name]);
+    }
+  };
+  const isSelected = (name: string): boolean => value.includes(name);
+
+  return (
+    <BaseBox height="100%">
+      <ListView>
+        <ListViewFilters
+          quickFilters={
+            <QuickFilterGroup selectionType="single" defaultValue="Captured">
+              <QuickFilter
+                title="Captured"
+                value="Captured"
+                trailing={<Counter value={234} color="positive" />}
+              />
+              <QuickFilter
+                title="Failed"
+                value="Failed"
+                trailing={<Counter value={234} color="negative" />}
+              />
+              <QuickFilter
+                title="Pending"
+                value="Pending"
+                trailing={<Counter value={234} color="neutral" />}
+              />
+            </QuickFilterGroup>
+          }
+          onSearchChange={(value) => console.log(value)}
+          numberOfSelectedFilters={2}
```

Comment:
We might be able to get this internally if we turn ListViewFilters into context and pass on this info from QuickFilter right 🤔